task_12/style1.css

```diff
@@ -0,0 +1,91 @@
+body {
+    margin: 0;
+   
+  }
+.menu{
+    display: grid;
+      grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+      grid-gap: 10px;
+      grid-auto-flow: dense;
+      padding: 10px;
+}
+
+
+.menu_container{
+    background-color: white;
+    border-radius: 3px;
+    box-shadow: 1px 2px 3px 2px rgba(0,0,0,0.15);
+    position: relative;
+    overflow: hidden;
+    padding: 7px;
+}
+  .menu_container_image{
+        
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      height: 200px;
+      background-color: rgba(146, 143, 143, 0.158);
+  }
+ 
+    .menu_container_image > img {
+      max-width: 100%;
+      max-height: 100%;
+    }
+
+    .menu_container_2grow{
```

Comment:
Why do you repeat all styles from .menu_container ?)
Incorrect name menu__container_grow_2